101-media-services-create/azuredeploy.json

```diff
@@ -0,0 +1,44 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "mediaServiceName": {
+      "type": "string",
+      "metadata": {
+        "description": "Name of the Media Services account. A Media Services account name is globally unique, all lowercase numbers or letters with no spaces."
+      }
+    }
+  },
+  "variables": {
+    "storageName": "[concat('storage', uniqueString(resourceGroup().id))]",
+    "storageType": "Standard_LRS"
+  },
+  "resources": [
+    {
+      "name": "[parameters('mediaServiceName')]",
+      "type": "Microsoft.Media/mediaServices",
+      "apiVersion": "2015-10-01",
+      "location": "[resourceGroup().location]",
+      "dependsOn": [
+        "[concat('microsoft.storage/storageaccounts/', variables('storageName'))]"
+      ],
+      "properties": {
+        "storageAccounts": [
+          {
+            "isPrimary": true,
+            "id": "[resourceId('microsoft.storage/storageaccounts/', variables('storageName'))]"
+          }
+        ]
+      }
+    },
+    {
+      "name": "[variables('storageName')]",
+      "type": "Microsoft.Storage/storageAccounts",
+      "apiVersion": "2015-06-15",
```

Comment:
True the absolute latest would be 2016-12-01, but no schema for that version has been published yet. In the schema repo (https://github.com/Azure/azure-resource-manager-schemas/tree/master/schemas) the latest available schema is 2016-01-01. We typically recommend using the latest apiVersion for which a schema is published, to ensure the best experience with JSON schema validation for people authoring templates in a JSON editor.